etc/service/rsyslog/run

```diff
@@ -6,4 +6,4 @@ rm -f /var/run/rsyslogd.pid
 mkdir -p /var/log/postfix
 chown syslog:syslog /var/log/postfix
 
-exec /usr/sbin/rsyslogd -n -c5 2>&1
```

Comment:
Why this change?

Reply:
`-c` parameter has been removed from rsyslogd ... on ubuntu based image it shows warning that this parameter will be ignored. But the alpine based image treats it as error, probably because it has much newer versions of the binaries.

Reply:
Sounds good.